cmd/minishift/cmd/addon/addon.go

```diff
@@ -23,15 +23,15 @@ import (
 const (
 	addOnConfigKey = "addons"
 
-	emptyAddOnError = "You must specify an add-on name. Use `minishift addons list` to view installed add-ons."
+	emptyAddOnError = "You must specify an add-on name. Run `minishift addons list` to view installed add-ons."
 	noAddOnMessage  = "No add-on with the name %s is installed."
 )
 
 var AddonsCmd = &cobra.Command{
 	Use:     "addons SUBCOMMAND [flags]",
 	Aliases: []string{"addon"},
-	Short:   "Manages Minishift add-ons",
-	Long:    "Manages Minishift add-ons. You can install, list, enable or disable Minishift add-ons.",
+	Short:   "Manages Minishift add-ons using sub-commands.",
```

Comment:
> IMO we do not need to mention sub-commands because the context is correct when we say Manages Minishift add-ons. Anyway the user will see the sub-commands in the usage of minishift addons --help.

IMO it gives valuable context. I can look at `minishift -h` and see directly which commands have subcommands. But I am ok, with not having it.